src/wikitextprocessor/data/en/namespaces.json

```diff
@@ -18,6 +20,7 @@
   "Main": {
     "id": 0,
     "name": "Main",
+    "subpages": false,
```

Comment:
Might be little confusing, I guess they consider "Template:a/doc" a subpage but not "eye/translations".